apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -55,6 +56,10 @@ export default class CoreLibrary {
         this.p5.drawSprites();
         this.drawVariableBubbles();
         this.drawSpeechBubbles();
+        // Don't show foreground effect in preview
+        if (!this.isPreviewFrame()) {
+          this.foregroundEffects.forEach(effect => effect.func());
+        }
```

Comment:
Similar to https://github.com/code-dot-org/code-dot-org/blob/mike/effects-for-spritelab/apps/src/p5lab/poetry/PoetryLibrary.js#L90-L92

This needs to live in both places because each lab has its own `executeDrawLoopAndCallbacks` function.